eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -276,29 +276,52 @@ private void clusterSubmittedVariants(List<? extends SubmittedVariantEntity> sub
                                                               SubmittedVariantEntity.class);
         BulkOperations dbsnpBulkOperations = mongoTemplate.bulkOps(BulkOperations.BulkMode.UNORDERED,
                                                                    DbsnpSubmittedVariantEntity.class);
+        BulkOperations bulkHistoryOperations = mongoTemplate.bulkOps(BulkOperations.BulkMode.UNORDERED,
+                SubmittedVariantOperationEntity.class);
+        BulkOperations dbsnpBulkHistoryOperations = mongoTemplate.bulkOps(BulkOperations.BulkMode.UNORDERED,
+                DbsnpSubmittedVariantOperationEntity.class);
         long numUpdates = 0;
         long numDbsnpUpdates = 0;
         for (SubmittedVariantEntity submittedVariantEntity : submittedVariantEntities) {
             if (submittedVariantEntity.getClusteredVariantAccession() != null) {
                 // no need to update the rs of a submitted variant that already has the correct rs
                 continue;
             }
-            Query query = query(where("_id").is(submittedVariantEntity.getId()));
-            Update update = new Update();
-            update.set(RS_KEY, getClusteredVariantAccession(submittedVariantEntity));
+
+            long rsid = getClusteredVariantAccession(submittedVariantEntity);
+            // Query to update the RSid in submittedVariantEntity
+            Query rsQuery = query(where("_id").is(submittedVariantEntity.getId()));
```

Comment:
minor detail: rsQuery is a bit confusing here. we are querying SS ids by SS hash. rsUpdate I guess is ok because we update the rs of a SS ID.

Reply:
Changed the variable name